### PR TITLE
Adds bolting and eletrifying logs to doors

### PR DESCRIPTION
**Pull request by Kmc2000**
:cl: Kmc2000
admin: Electrifying and bolting doors is now logged.
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
tested and shit idk lol

**Comment by DLCabbose** [comment]
lol

nice gamer

**Comment by Takahiru** [comment]
more logs is always good

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -94,6 +94,8 @@
 
 	var/air_tight = FALSE	//TRUE means density will be set as soon as the door begins to close
 	var/prying_so_hard = FALSE
+	var/list/Bolt_log = list() //Who can it be bolting all my doors? Go away, don't come down here no more.
+	var/list/Shocking_log = list() //yogs - who electrified this door.
```
Comment (ThatLing) [inline code/game/machinery/doors/airlock.dm]: Define the list in /Initialize()

Comment (ThatLing) [inline code/game/machinery/doors/airlock.dm]: // yogs start
// yogs end

**Review by ThatLing — DISMISSED** [review]

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
you have to put //yogs on every line you change here

**Review by ThatLing — APPROVED** [review]

**Pull request merged**